br/pkg/lightning/importer/dup_detect.go

```diff
@@ -368,3 +398,18 @@ func simplifyTable(
 	}
 	return newTblInfo, newColPerm
 }
+
+const conflictOnHandle = int64(-1)
+
+func decodeIndexID(key []byte) (int64, error) {
+	switch {
+	case tablecodec.IsRecordKey(key):
```

Comment:
Does serverless TiDB have a different codec?

Reply:
keyspace is added only when transmit data to TiKV, TiDB is agnostic about it